lib/category/models/category.dart

```diff
@@ -8,7 +9,7 @@ class Category {
   final String code;
   final String name;
   final String? destination;
-  final CategoryImage? image;
+  final StrapiImage? image;
```

Comment:
Hmm this is a "committing to Strapi" type change. Let's discuss future options here.